APE_specutil.rst

```diff
@@ -0,0 +1,481 @@
+Vision for Astropy Spectroscopic Tools
+======================================
+
+author: S. M. Crawford, N. Earl, A. Ginsburg, E. Tollerud
+
+date-created: 2017 April 21 <replace with the date you submit the APE>
+
+date-last-revised: 2017 Jul 19 <keep this up to date anytime something changes>
+
+type:  Process, Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+Spectroscopy lies at the core of much of astronomy.  The goal of this APE is to
```

Comment:
I think this first sentence can be removed since it's repeated in the detailed description.